===== Patch instructions generated by SAVER =====
[+] { Insert tmp := main:c->daemonize at 20 (line 241, column 2) and replace the original expression at 6 (line 247, column 6) under { main:n$21 == { 1 } },
        Insert tmp := main:c->error_num at 20 (line 241, column 2) and replace the original expression at 10 (line 250, column 7) under { main:n$21 == { 1 } } }

===== Actual patch in diff format =====
@@ -239,15 +239,17 @@ int main(int argc, char *argv[])
 
 	c->daemonize = my_args.daemonize == 1;
 	bret = c->start(c, 1, my_args.argv);
+	tmp1 = c->daemonize;
+	tmp2 = c->error_num;
 	lxc_container_put(c);
 	if (!bret) {
 		fprintf(stderr, "Failed run an application inside container\n");
 		exit(EXIT_FAILURE);
 	}
-	if (c->daemonize)
+	if (tmp1)
 		exit(EXIT_SUCCESS);
 	else {
-		if ((c->error_num)) {
+		if (tmp2) {
 			exit((c->error_num));
 		} else {
 			/* Try to die with the same signal the task did. */
